am/README.adoc

```diff
@@ -183,6 +183,34 @@ replicaset] for detailed documentation on helm chart.
 
 |api.http.services.core.service.enabled |bool |`+false+` |
 
+|api.http.host |string |`+"0.0.0.0"+` |
+
+|api.http.port |bool |`+8093+` |
+
+|api.http.idleTimeout |int |`+30000+` |
+
+|api.http.acceptors |int |`+-1+` |
+
+|api.http.selectors |int |`+-1+` |
+
+|api.http.outputBufferSize |int |`+32768+` |
+
+|api.http.requestHeaderSize |int |`+8192+` |
+
+|api.http.responseHeaderSize |int |`+8192+` |
+
+|api.http.pool.minThreads |int |`+8192+` |
+
+|api.http.pool.maxThreads |int |`+8192+` |
+
+|api.http.pool.idleTimeout |int |`+8192+` |
+
+|api.http.pool.queueSize |int |`+8192+` |
```

Comment:
I'm pretty sure that these are not the real default values 🙂

Reply:
FP